modules/features/settings/src/main/java/au/com/shiftyjelly/pocketcasts/settings/HelpFragment.kt

```diff
@@ -90,16 +101,29 @@ class HelpFragment : BaseFragment(), HasBackstack, Toolbar.OnMenuItemClickListen
     }
 
     @SuppressLint("SetJavaScriptEnabled")
-    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
         val view = inflater.inflate(VR.layout.fragment_webview, container, false)
 
         webView = (view.findViewById<View>(VR.id.webview) as WebView).apply {
             webViewClient = SupportWebViewClient()
             loadUrl(loadedUrl ?: Settings.INFO_FAQ_URL)
             settings.javaScriptEnabled = true
             settings.textZoom = 100
+            //disables horizontal scrolling bar not scroll
+            isHorizontalScrollBarEnabled = false
+            settings.layoutAlgorithm = TEXT_AUTOSIZING
             settings.domStorageEnabled = true
+            // Will allow wide viewport as much device width has for responsiveness
+            settings.useWideViewPort = true
```

Comment:
I wasn't able to tell what difference this made on the Help & Feedback screen. Can help me understand why this is needed?